python/ray/air/_internal/util.py

```diff
@@ -35,7 +36,13 @@ class StartTraceback(Exception):
 
 
 def skip_exceptions(exc: Optional[Exception]) -> Exception:
-    """Skip all contained `StartTracebacks` to reduce traceback output"""
+    """Skip all contained `StartTracebacks` to reduce traceback output.
```

Comment:
Any ideas on why  Maximum recursion happens iff `pickling_support.install()` is called?